sdks/python/apache_beam/runners/interactive/pipeline_instrument.py

```diff
@@ -80,7 +80,10 @@ def __init__(self, pipeline, options=None):
     # A mapping from PCollection id to python id() value in user defined
     # pipeline instance.
     (self._pcoll_version_map,
-     self._cacheables) = cacheables(self.pcolls_to_pcoll_id)
+     self._cacheables,
```

Comment:
Adding comment here since GitHub does not allow comments on unchanged lines.
It's probably not obvious what it means to "instrument" a pipeline. From my context back when working on Interactive Beam, I would guess it's something similar to the original pipeline_analyzer, (and I have to admit that "analyze" pipeline was not a much better.), but it's still fuzzy.

Maybe adding some comment summarizing what pipeline_instrument does would be good. In a separate PR

Reply:
Yes, I agree! It's similar to what pipeline_analyzer was doing but using `PipelineVisitor` and some `globals` instead of pipeline protos. We haven't achieved a consensus about what this should be called. We also have `cache augmented pipeline` and etc.. But none of the names describe what the module does accurately because the implementation details cover so many different execution routes and scenarios.
We'll definitely get back to the documentation here once we make the final naming decision. Thanks!